pkg/services/store/object/summary.go

```diff
@@ -24,6 +24,20 @@ type ObjectSummary struct {
 	_ interface{}
 }
 
+// Reference to another object outside itself
+// This message is derived from the object body and can be used to search for references.
+// This does not represent a method to declare a reference to another object.
+type ExternalReference struct {
```

Comment:
this is unused

Reply:
🤔 it is unused in protobuf... but used in the summary

(the code that touches it is in search not here)

Reply:
ohh.. it used to be in `services/store/object/object.go`, it was just moved here 🤦